### ISSUE TITLE
nvidia: 460.80 breaks on some displayport monitors

### ISSUE THREAD

**Issue by kiwec** [void-linux/void-packages#30952]
### System

* xuname: Void 5.8.6-lqx_1 x86_64 AuthenticAMD notuptodate rrrmDFFFFFFFFF
* package: nvidia-460.80

### Expected behavior

`startx` starts an x session

### Actual behavior

System freezes

### Steps to reproduce the behavior

1. Install latest nvidia package

See https://github.com/void-linux/void-packages/pull/30809

2. Get one of the affected monitors (sorry, even nvidia devs have trouble reproducing)

https://forums.developer.nvidia.com/t/465-24-02-page-fault/175782

**Comment by kiwec** [comment]
If you're affected, unplugging displayport monitors will fix it temporarily.

To downgrade to a working version, I had to do the following : 

```
xdowngrade /var/cache/xbps/nvidia-460.56_1.x86_64.xbps /var/cache/xbps/nvidia-libs-460.56_1.x86_64.xbps /var/cache/xbps/nvidia-gtklibs-460.56_1.x86_64.xbps /var/cache/xbps/nvidia-dkms-460.56_1.x86_64.xbps /var/cache/xbps/nvidia-libs-32bit-460.56_1.x86_64.xbps /var/cache/xbps/nvidia-opencl-460.56_1.x86_64.xbps
dkms uninstall -m nvidia/460.56 -c void-packages/srcpkgs/nvidia/files/dkms.conf --force
dkms install -m nvidia/460.56 -c void-packages/srcpkgs/nvidia/files/dkms.conf --force
```

**Comment by NeelChotai** [comment]
This should be fixed in the 470 drivers, anyone with this issue should downgrade as suggested above and `xbps-pkgdb -m hold` installed `nvidia` and `nvidia-*` packages until then.

**Comment by ahesford** [comment]
Is this issue resolved with newer nvidia packages?

**Comment by kiwec** [comment]
> Is this issue resolved with newer nvidia packages?

It is (according to the nvidia forums) fixed in 470.42.01, but no one made a PR to update the nvidia package yet.

**Comment by kiwec** [comment]
Fixed in https://github.com/void-linux/void-packages/pull/32100
